wiki_articles/learn_cpp.md

```diff
@@ -1,27 +1,30 @@
 # How to Learn C++ Programming
 
-Everyone learns differently, but the best way is to learn by doing:
-Working on personal projects, solving programming challenges, etc.
-Here are some resources to accompany you:
+We generally recommend a good book to learn the necessary fundamentals:
+- [Murach’s C++ Programming](https://www.murach.com/shop/murach-s-c-programming-2nd-edition-detail)
+- [Programming: Principles and Practice Using C++](https://stroustrup.com/programming.html)
+- [learncpp.com](https://www.learncpp.com/)
+
+To actually write and run C++ code, you will need a compiler, editor, and debugger.
+We *strongly* recommend to start out using an IDE, which will provide all these tools for you:
 
 <!-- inline -->
-## C++ Tutorials
-- [Programiz](https://www.programiz.com/cpp-programming)
-- [codecademy](https://www.codecademy.com/learn/learn-c-plus-plus)
-- [learncpp.com](https://www.learncpp.com/) if you have prior experience with programming
+## :microsoft: Windows
+- [Visual Studio](https://discord.com/channels/331718482485837825/1165492293810257920/1165493161242673172)
+- [CLion](https://www.jetbrains.com/clion/)
 
 <!-- inline -->
-## Practice
-- [Programming Challenges](https://discord.com/channels/331718482485837825/574580408407293954/668548805779652649)
-- [CodinGame](https://www.codingame.com)
-- [LeetCode](https://leetcode.com/)
-- [HackerRank](https://www.hackerrank.com/)
+## :tux: Linux
+- [Qt Creator](https://www.qt.io/product/development-tools)
+- [Visual Studio Code](https://code.visualstudio.com/docs/cpp/config-linux)
+- [CLion](https://www.jetbrains.com/clion/)
 
 <!-- inline -->
-## Books
-<:stackoverflow:1074747016644661258>
-[The Definitive C++ Book Guide and List](https://stackoverflow.com/a/388282/5740428)
```

Comment:
Do you want to make note of this in a see also section or just leave it off?

Reply:
I went through that list and I don't think there's much value in linking a beginner to it in addition to the recommendations that we already put there.

Reply:
As I said on discord, what I was trying to do here is to provide beginners with a clear path rather than a sea of options to pick from.